foxy/distribution.yaml

```diff
@@ -1821,6 +1821,17 @@ repositories:
       url: https://github.com/ros2/ros2cli.git
       version: foxy
     status: maintained
+  ros2cli_common_extensions:
+    doc:
+      type: git
+      url: https://github.com/ros2/ros2cli_common_extensions.git
+      version: master
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/ros2-gbp/ros2cli_common_extensions-release.git
+      version: 0.1.0-1
+    status: maintained
```

Comment:
I think it doesn't hurt to add a `source` entry. Also, consider enabling pull request testing (it can add as a sanity check when adding/removing packages from the metapackage).

Reply:
Done, I've also changed the `version` to `foxy` in the doc entry.
`foxy` and `master` branch are currently synced (I created foxy branch after doing the release).